src/playground/render-gui.jsx

```diff
@@ -37,5 +37,10 @@ export default appTarget => {
         window.onbeforeunload = () => true;
     }
 
-    ReactDOM.render(<WrappedGui backpackOptions={backpackOptions} />, appTarget);
+    ReactDOM.render(
+        <WrappedGui
+            canCreateNew
```

Comment:
Don't need this anymore.

Reply:
It sort of depends on your definition of "need" :)

Isn't it better to explicitly set this, because we want it, rather than assume the default is what we want it to be?

Reply:
Thinking about it more, I think we definitely want the playground to use the defaults, since when you're developing, you don't expect any props to be set, and you expect all the defaults to be used. It would be a frustrating experience to try changing that default and not seeing anything happen, only to find that there is this one prop set at the playground level.

Reply:
Removed, to make usage clearer